Framework/DataHandling/src/LoadBankFromDiskTask.cpp

```diff
@@ -222,10 +235,12 @@ LoadBankFromDiskTask::loadEventId(::NeXus::File &file) {
     m_max_id =
         *(std::max_element(event_id.get(), event_id.get() + m_loadSize[0]));
 
-    if (m_min_id > static_cast<uint32_t>(m_loader.eventid_max)) {
-      // All the detector IDs in the bank are higher than the highest 'known'
-      // (from the IDF)
-      // ID. Setting this will abort the loading of the bank.
+    auto offset_min_id = m_min_id + m_loader.pixelID_to_wi_offset;
+    if (offset_min_id >=
```

Comment:
This addition seems to be responsible for the test failure LoadEventNexusTest (run_MPI_load)